presto-spi/src/main/java/com/facebook/presto/spi/TableNotFoundException.java

```diff
@@ -19,11 +19,13 @@
 public class TableNotFoundException
         extends NotFoundException
 {
+    public static final String TABLE_NOT_FOUND_FMT_STRING = "Table '%s' not found";
```

Comment:
nit: the convention in Presto code base doesn't usually do abbreviation . So maybe just `TABLE_NOT_FOUND_FORMAT_STRING`

Reply:
Changed it